metadata_test.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"testing"
+)
+
+func TestGithubRegexp(t *testing.T) {
+	cases := map[string]string{
+		`git https://github.com/paultag/go-config">`: "paultag/go-config",
+		`https://github.com/owner/repo->`:            "owner/repo-",
```

Comment:
Where did you see this in practice? Do you have a URL where I can find such a tag?